src/frontend/screens/ObservationEdit/LocationView.tsx

```diff
@@ -19,66 +19,95 @@ const m = defineMessages({
 });
 
 export const LocationView = () => {
-  const liveLocation = useMostAccurateLocationForObservation();
-  const observationValue = usePersistedDraftObservation(
-    observationValueSelector,
+  const observationValue = usePersistedDraftObservation(store => store.value);
+
+  return observationValue?.metadata.manualLocation ? (
+    <LocationViewManualPosition />
+  ) : (
+    <LocationViewUpdatePosition />
+  );
+};
+
+const LocationViewManualPosition = () => {
+  const observationValue = usePersistedDraftObservation(store => store.value);
+  return (
+    <LocationViewInner
+      lat={observationValue?.lat}
+      lon={observationValue?.lon}
+      accuracy={null}
+    />
   );
-  const coordinateFormat = usePersistedSettings(coordinateFormatSelector);
+};
+
+const LocationViewUpdatePosition = () => {
+  const observationValue = usePersistedDraftObservation(store => store.value);
+  //only update the location if the accuracy increases OR the user has moved outside of the accuracy's radius of uncertainty.
+  const {location} = useLocation(
+    ({accuracy}) => accuracy === 'better' || accuracy === 'newBounds',
+  );
+  const {updateObservationPosition} = useDraftObservation();
+
+  useEffect(() => {
+    const newCoord = !location
+      ? undefined
+      : Object.entries(location.coords).map(
+          ([key, val]) => [key, val === null ? undefined : val] as const,
+        );
 
-  const coordinateInfo = observationValue?.metadata.manualLocation
-    ? {
-        lat: observationValue.lat,
-        lon: observationValue.lon,
-        accuracy: liveLocation?.coords?.accuracy,
-      }
-    : {
-        lat: liveLocation?.coords?.latitude,
-        lon: liveLocation?.coords?.longitude,
-        accuracy: liveLocation?.coords?.accuracy,
-      };
+    updateObservationPosition({
+      position: {
```

Comment:
Previously (Mapeo legacy) we had been saving all the location information from location state to the observation. Is there a reason to stop saving this info?

Reply:
Will be addressed in [this](https://github.com/digidem/comapeo-mobile/issues/402)